aries-site/src/pages/components/grid.mdx

```diff
@@ -13,3 +22,201 @@ import { GridExample } from '../../examples';
 >
   <GridExample />
 </Example>
+
+## Getting to know Grid
+Grid is a powerful tool. It provides tremendous flexibility and control enabling designers and 
+developers at HPE to create optimal user experiences at any screen size or resolution!
+
+With all of Grid's power, there is a bit of a learning curve. The following introduces its key 
+concepts and will accelerate your command of the tool.
+
+### Background
+Based on [CSS Grid Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout), 
+Grommet's Grid simplifies and presents the API in a manner consistent with other components 
+in the Grommet library.
+
+Grid is composed of columns and rows and its properties allow for:
+- Fixed column widths and row heights, when use cases desire precise dimensions for grid cells.
+- Flexible column widths and heights, allowing grid cell contents to gracefully respond across 
+a variety of browser widths.
+- Fluid grids allowing columns and rows to grow and adapt to the content provided.
+- Precise placement of elements within the grid via [areas](#areas).
+
+## Grid Properties & Key Concepts
+Grid's primary properties include: [columns](#columns), [rows](#rows), and [areas](#areas). The 
+following examples will demonstrate how these properties may be applied to affect 
+[content layouts](/templates/content-layouts).
+
+### Columns
+Columns may be specified as [uniform widths](#uniform-columns), [mixed widths](#mixed-width-columns), 
+or as [min-max ranges](#flexible-columns) which allow the contents to flex gracefully.
+
+#### Uniform Columns
+Uniform columns are ideal for presenting a collection of like items such as a grid of images, 
+a catalog of services, or a [dashboard of cards](/templates/dashboards#three-column-dashboard).
+
+Specifying the column width as single string will repeat multiple columns of that size, as long as 
+there is room for more. For example, `columns="medium"` will fit as many "medium" columns into the 
+grid as the overall width allows. Within a wide screen, this might allow 3 or 4 columns; within a narrow 
+screen, maybe only 1 or 2.
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/UniformColumns.js"
+  width={{ min: 'medium' }}
+>
+  <UniformColumns />
+</Example>
+
+#### Mixed-Width Columns
+Mixed-width columns allow for control over layouts desiring columns of various widths to appropriately present 
```

Comment:
Having a hard time following this sentence. Maybe break it up into:

"Mixed-width columns allow for control over layouts desiring columns of various widths. This can help to appropriately present different content types or establish content hierarchy with wider columns for primary content and narrower for secondary content such as an [activity feed](https://deploy-preview-2423--keen-mayer-a86c8b.netlify.app/templates/dashboards#two-column-dashboard).

Reply:
Great callout. Thank you for the suggestion. Done.